src/live_restore/live_restore_fs.c

```diff
@@ -851,23 +851,22 @@ __live_restore_fill_hole(WT_FILE_HANDLE *fh, WT_SESSION *wt_session, WT_TIMER *s
      * shrinking the hole in the stack below us. This is why we always read from the start at the
      * beginning of the loop.
      */
-    char buf[WT_LIVE_RESTORE_READ_SIZE];
-    size_t read_size = WT_MIN(hole->len, (size_t)WT_LIVE_RESTORE_READ_SIZE);
+    size_t read_size = WT_MIN(hole->len, lr_fh->destination.back_pointer->read_size);
     uint64_t time_diff_ms;
 
-    __wt_verbose_debug3(session, WT_VERB_LIVE_RESTORE,
+    __wt_timer_evaluate_ms(session, start_timer, &time_diff_ms);
+    __wt_verbose_debug3(session, WT_VERB_FILEOPS,
```

Comment:
Reset this to WT_VERB_LIVE_RESTORE. We want to move away and not use FILEOPS in here.

Reply:
[79ace73](https://github.com/wiredtiger/wiredtiger/pull/11430/commits/79ace73cb962fb5a6c60e99789a50aabc4ccf712)